docs/standard/choosing-core-framework-server.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Choosing between .NET Core and .NET Framework for server apps
+description: A guide on which flavor of .NET you should consider when building a server app in .NET.
+keywords: .NET, .NET Core, .NET Framework
+author: cartermp
+manager: wpickett
+ms.author: phcart
+ms.date: 11/16/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 155553e4-89a2-418d-be88-4e75f6c3cc69
+---
+
+# Choosing between .NET Core and .NET Framework for your server apps
+
+There are two supported choices of runtime for building server-side applications with .NET: .NET Framework and .NET Core. Both share a lot of the same .NET platform components and you can share code across the two. However, there are fundamental differences between the two and your choice will depend on what you want to accomplish.  This article provides guidance on when to use each.
+
+You should use .NET Core for your server application when:
+
+* You have cross-platform needs.
+* You are targeting microservices.
+* You are using Docker containers.
+* You need high performance and scalable systems.
+* You need side by side of .NET versions by application.
+
+You should use .NET Framework for your server application when:
+
+* Your application currently uses .NET Framework (recommendation is to extend instead of migrate)
+* You need to use third party .NET libraries or NuGet packages not available for .NET Core.
+* You need to use .NET technologies that are not available for .NET Core.
+* You need to use a platform that doesn’t support .NET Core.
+
+## When to choose .NET Core
+
+The following is a detailed set of important reasons why you would care and want to use .NET Core.
+
+### Cross-Platform needs
+
+Clearly, if your goal is to have an application (web/service) that should be able to run across platforms (Windows, Linux and MacOS), the best choice is to use .NET Core.
+
+.NET Core also supports any OS as your development workstation. Visual Studio provides a IDE for Windows, but you can also use Visual Studio Code on MacOS, Linux and Windows which fully support .NET Core as well, including intellisense and debugging. You can also target .NET Core with most third party editors like Sublime, Emacs, VI and can get editor intellisense using the open source [Omnisharp](http://www.omnisharp.net/) project. You can also avoid any code editor and directly use the .NET Core command-line tools, available for any platform.
```

Comment:
intellisense -> IntelliSense (multiple instances)

.NET Core also supports any OS as your development workstation -> kind of a broad statement. Shouldn't this say ".NET Core also supports any of those platforms as your development workstation"

a IDE -> an integrated development environment (IDE)

fully support -> fully supports (are you saying VS supports) and remove as well

third party -> third-party

any platform -> all supported platforms